go.mod

```diff
@@ -14,12 +16,13 @@ require (
 	github.com/vektra/errors v0.0.0-20140903201135-c64d83aba85a
 	golang.org/x/net v0.0.0-20210330142815-c8897c278d10
 	golang.org/x/text v0.3.5 // indirect
-	google.golang.org/grpc v1.27.1
-	k8s.io/api v0.20.5
-	k8s.io/apimachinery v0.20.5
-	k8s.io/client-go v0.20.5
-	k8s.io/klog/v2 v2.4.0
-	k8s.io/kubernetes v1.20.5
+	google.golang.org/grpc v1.35.0
+	google.golang.org/protobuf v1.25.0
+	k8s.io/api v0.21.0
+	k8s.io/apimachinery v0.21.0
+	k8s.io/client-go v0.21.0
+	k8s.io/klog/v2 v2.8.0
+	k8s.io/kubernetes v1.21.0
```

Comment:
I did a separate PR updating to kubernetes v1.21.2. I've found it safest and hassle-free to keep grpc version in sync with k/k

Reply:
Right, I rebased my PR on the master so this has been addressed.